crates/noirc_driver/src/lib.rs

```diff
@@ -189,7 +191,12 @@ impl Driver {
         let ast = monomorphise(main_function, self.context.def_interner);
 
         // Compile Program
-        let circuit = match create_circuit(ast, np_language, show_ssa) {
+        let circuit = match create_circuit(
+            ast,
+            np_language.clone(),
```

Comment:
cloning here is unnecessary

Reply:
It's needed because the next line uses `np_language` and Language is not Copyable.

The solutions are to make language Copyable or make that function take a reference. I'll modify the function to take a reference since it does not need to own it

Reply:
I had missed the usage on the line below somehow, so this cloning is fine. Taking it in by value is also fine since you do end up needing the value twice. I haven't looked into it, but if `acvm::default_is_blackbox_supported` doesn't really need to take np_language by value then we can remove the cloning altogether. I am fine with it as is though, there are plenty of more egregious examples to fix beforehand if we're worried about cloning optimizations.